packages/admin-ui/src/lib/marketing/src/components/promotion-detail/promotion-detail.component.html

```diff
@@ -92,6 +92,19 @@
                             formControlName="perCustomerUsageLimit"
                         />
                     </vdr-form-field>
+                    <vdr-form-field
+                        [label]="'marketing.usage-limit' | translate"
+                        for="usageLimit"
```

Comment:
Can we get a tooltip here where we can have a short explanation?

```
[tooltip]="'marketing.usage-limit-tooltip'"
```

We could also have a tooltip for `marketing.per-customer-limit` in the preceding form field too - this allows us to clearly describe the difference between these 2 similar fields in the UI.